bin/import_slr57.py

```diff
@@ -0,0 +1,221 @@
+    for i, _ in enumerate(pool.imap_unordered(one_sample, samples), start=1):
+        bar.update(i)
+    bar.update(num_samples)
+    pool.close()
+    pool.join()
+
+    with open(target_csv_template.format('train'), 'w') as train_csv_file:  # 80%
+        with open(target_csv_template.format('dev'), 'w') as dev_csv_file:  # 10%
+            with open(target_csv_template.format('test'), 'w') as test_csv_file:  # 10%
+                train_writer = csv.DictWriter(train_csv_file, fieldnames=FIELDNAMES)
+                train_writer.writeheader()
+                dev_writer = csv.DictWriter(dev_csv_file, fieldnames=FIELDNAMES)
+                dev_writer.writeheader()
+                test_writer = csv.DictWriter(test_csv_file, fieldnames=FIELDNAMES)
+                test_writer.writeheader()
+
+                for i, item in enumerate(rows):
+                    transcript = validate_label(item[2])
+                    if not transcript:
+                        continue
+                    wav_filename = item[0]
+                    i_mod = i % 10
+                    if i_mod == 0:
+                        writer = test_writer
+                    elif i_mod == 1:
+                        writer = dev_writer
+                    else:
+                        writer = train_writer
+                    writer.writerow(dict(
+                        wav_filename=wav_filename,
+                        wav_filesize=os.path.getsize(wav_filename),
+                        transcript=transcript,
+                    ))
+
+    print('Imported %d samples.' % (counter['all'] - counter['failed'] - counter['too_short'] - counter['too_long']))
+    if counter['failed'] > 0:
+        print('Skipped %d samples that failed upon conversion.' % counter['failed'])
+    if counter['invalid_label'] > 0:
+        print('Skipped %d samples that failed on transcript validation.' % counter['invalid_label'])
+    if counter['too_short'] > 0:
+        print('Skipped %d samples that were too short to match the transcript.' % counter['too_short'])
+    if counter['too_long'] > 0:
+        print('Skipped %d samples that were longer than %d seconds.' % (counter['too_long'], MAX_SECS))
+    print('Final amount of imported audio: %s.' % secs_to_hours(counter['total_time'] / SAMPLE_RATE))
+
+def handle_args():
+    parser = argparse.ArgumentParser(description='Importer for African Accented French dataset. More information on http://www.openslr.org/57/.')
+    parser.add_argument(dest='target_dir')
+    parser.add_argument('--filter_alphabet', help='Exclude samples with characters not in provided alphabet')
+    parser.add_argument('--normalize', action='store_true', help='Converts diacritic characters to their base ones')
+    return parser.parse_args()
+
+if __name__ == "__main__":
+    CLI_ARGS = handle_args()
+    ALPHABET = Alphabet(CLI_ARGS.filter_alphabet) if CLI_ARGS.filter_alphabet else None
+
+    def label_filter(label):
```

Comment:
I think this should be moved from the main-if to top-level as it only uses global vars. It's just a bit confusing having it defined inside an if-statement...